docs/tasks/README.md

```diff
@@ -3,6 +3,23 @@
 This section contains pages that show how to perform individual tasks.
 A task page shows how to do an operation with a short sequence of steps.
```

Comment:
The following pages show you how to perform individual tasks based on two type of roles:
* Batch administrator
* Batch user